corehq/apps/users/models.py

```diff
@@ -1088,13 +1088,19 @@ def sync_to_django_user(self):
             django_user = User(username=self.username)
         for attr in DjangoUserMixin.ATTRS:
             attr_val = getattr(self, attr)
-            if not attr_val and attr != 'last_login':
+            if attr in [
+                'is_active',
+                'is_staff',
+                'is_superuser',
+            ]:
+                attr_val = bool(attr_val)
```

Comment:
in the tests it was getting passed ```''```.  had something to do with the couch-sql integration for users.  In django <1.10 it worked, but in 1.10 you have to assign boolean values to these fields.